package.json

```diff
@@ -21,6 +23,12 @@
     "docusaurus": "^1.0.15",
     "husky": "^0.14.3",
     "precise-commits": "^1.0.2",
-    "prettier": "^1.12.1"
+    "prettier": "^1.12.1",
+    "copyfiles": "1.2.0",
+    "workbox-cli": "^3.0.1"      
+  },
+  "dependencies": {
+    "express": "4.15.4",
```

Comment:
Why are we running a server? Isn't the whole point of the project to go serverless?

Reply:
I removed it

Reply:
As far as I can tell, it is still there.

Reply:
Which line are you referring to?

Reply:
express dependency